src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -97,6 +97,8 @@ public function denormalize($data, $type, $format = null, array $context = [])
         $dateTimeFormat = $context[self::FORMAT_KEY] ?? null;
         $timezone = $this->getTimezone($context);
 
+        $data = \is_string($data) ? trim($data) : $data;
```

Comment:
IMHO we should add a test for eg. `" <date> "` also ...

Reply:
and then in case of a format, im not sure that's expected :) https://3v4l.org/dTZHB

Reply:
done for both cases